packages/salesforcedx-apex-debugger/src/messages/index.ts

```diff
@@ -21,17 +23,15 @@ function loadMessageBundle(config?: Config): Message {
       : `${BASE_FILE_NAME}.${locale}.${BASE_FILE_EXTENSION}`;
   }
 
-  const base = new Message(
-    require(`./${resolveFileName(DEFAULT_LOCALE)}`).messages
-  );
```

Comment:
this one was tricky as it didn't like using requires.  I worked around it since we only have two languages.